support-frontend/assets/components/directDebit/directDebitActions.js

```diff
@@ -68,18 +72,23 @@ function payDirectDebitClicked(): Function {
   return (dispatch: Function, getState: Function) => {
 
     const {
+      sortCodeString,
       sortCodeArray,
       accountNumber,
       accountHolderConfirmation,
     } = getState().page.directDebit;
 
-    const sortCode = sortCodeArray.join('');
+    const sortCode = sortCodeArray.join('') || sortCodeString;
     const isTestUser: boolean = getState().page.user.isTestUser || false;
     const { csrf } = getState().page;
 
     dispatch(resetDirectDebitFormError());
 
-    if (!accountHolderConfirmation) {
+    /* The new version of the direct debit form doesn't have an account holder confirmation checkbox,
+    so this if-statement is not needed for the new form. The old form uses sortCodeArray whereas the
+    new form uses sortCodeString, so checking if sortCodeString is empty is a way of checking which
+    version of the DD form it is and whether to do this check. */
+    if (!accountHolderConfirmation && sortCodeString.length === 0) {
```

Comment:
the note explains this piece of code nicely `sortCodeString.length === 0`, but if you change this to a name like `const isProgressiveDisclosureForm = sortCodeString.length > 0` and use it as `if (!accountHolderConfirmation && !isProgressiveDisclosureForm)`or something much better we can remove the note, because the variable explains the note

Reply:
Thanks - have done 👍